src/offset/local/mod.rs

```diff
@@ -145,8 +147,38 @@ impl Local {
     /// let now_with_offset = Local::now().with_timezone(&offset);
     /// ```
     pub fn now() -> DateTime<Local> {
+        #[cfg(all(feature = "test-override", test))]
+        if let Some(t) = OVERRIDE_NOW.with(|o| *o.borrow()) {
+            return t.into();
+        }
         Utc::now().with_timezone(&Local)
     }
+
+    /// Override the value that will be returned by `Local::now()` and `Utc::now()`, for use in
+    /// tests.
+    ///
+    /// This method is only available behind the `test-override` feature, only works within the
+    /// current thread, and only in `cfg(test)`.
+    ///
+    /// ```no_run
```

Comment:
Codecov correctly says the lines of this doctest are not covered. But I can't help it :smile:.